HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 History
 =======
 
+1.0.1 (2020-06-12)
+------------------
+
+New features:
+
+* New language detect function
```

Comment:
Hey @gavishpoddar, you shouldn't add this to the changelog. We will merge this PR and after that we will release a new version which will include this and more changes. :)